frontends/web/src/components/devices/bitbox02/checkbackup.tsx

```diff
@@ -31,21 +32,24 @@ type Props = CheckProps & TranslateProps;
 interface State {
     activeDialog: boolean;
     message: string;
+    foundBackup: Backup;
```

Comment:
`foundBackup?: Backup;` as it could be undefined/not found

Reply:
Good points! Fixed